src/benchmarks/micro/libraries/System.Runtime/Perf.Int128.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Tests
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class Perf_Int128
```

Comment:
Why only this subset of tests? Do we want to provide meaningful benchmarks for most of Int128/UInt128 since it is not a language primitive, but does provide the full set of functionality?

Reply:
> Why only this subset of tests

My main goal was to add `CopySign` benchmarks. I've realized that `Int128` has zero benchmarks, so I've copy-pasted `Int64` benchmarks and adjusted them. Just to have some benchmarks as a starting point